### DISCUSSION TITLE
False Phishing Alarm

### DISCUSSION THREAD

**Discussion by sohamc2001** [community/community#37232] [Other Feature Feedback, Questions, & Ideas] [answered]
Hello Github Support Team @github-staff,

I am Soham Chakraborty. I recently received an email from the GitHub support team about my GitHub pages site - https://sohamc2001.github.io/facebook-clone. It was written that an individual had contacted GitHub support about my site and reported it as a phishing site. I am aware of the fact that every browser shows my project as a phishing site. I don't know why this happens. I am an engineering student. This is just a personal project of mine to study HTML & CSS. My project is neither the official Facebook site nor it is a phishing site and nor does it collect any personal data. I made this Facebook clone project as a personal project of mine. I do not know why this happens. As advised in the GitHub mail I have already added a BOLD DISCLAIMER to my page. I have also added a readme disclaimer to my repository. Please advise me on what should I do! I am awaiting a reply. I am attaching the previous mail here so you can get context.

I am also attaching my GitHub profile and the repository link so that my code can be checked for phishing.

My gihub profile - https://github.com/sohamc2001
Repository link - https://github.com/sohamc2001/facebook-clone

Please help in any way you can so that this false alarm problem can be fixed.

Thank you,
Soham Chakraborty

[Gmail - [GitHub Support] - A note from GitHub about your GitHub Pages site.pdf](https://github.com/community/community/files/9862098/Gmail.-.GitHub.Support.-.A.note.from.GitHub.about.your.GitHub.Pages.site.pdf)

**Comment by queenofcorgis** [comment, marked as answer]
Hi there! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them as we are unable to assist in matters such as this. :) We'll be locking this thread so you can go through the proper channels!
